packages/cli/README.md

```diff
@@ -76,9 +76,9 @@ Options:
 
 Commands:
   create [options] [project-directory]  Create a Semaphore project with a supported template.
-  get-groups [options]                  Get the list of groups from a supported network (e.g. goerli or arbitrum).
-  get-group [options] [group-id]        Get the data of a group from a supported network (e.g. goerli or arbitrum).
-  get-members [options] [group-id]      Get the members of a group from a supported network (e.g. goerli or arbitrum).
-  get-proofs [options] [group-id]       Get the proofs of a group from a supported network (e.g. goerli or arbitrum).
+  get-groups [options]                  Get the list of groups from a supported network (e.g. sepolia or arbitrum).
```

Comment:
Shouldn't this be like

```suggestion
  get-groups [options]                  Get the list of groups from a supported network (e.g. sepolia or arbitrum-sepolia).
```

Reply:
I mean arbitrum one here!

Reply:
Ahhhh, got it. Are we supporting `arbitrum-one` too?

Reply:
Yes!

Reply:
Because I don't see it in the `networks.json` file.

Reply:
It's here: https://github.com/semaphore-protocol/semaphore/blob/feat/semaphore-v4/packages/contracts/deployed-contracts/arbitrum.json